samples/net/lwm2m_client/README.rst

```diff
@@ -90,7 +93,7 @@ To build the lwm2m-client sample for QEMU with DTLS support do the following:
    :zephyr-app: samples/net/lwm2m_client
    :host-os: unix
    :board: qemu_x86
-   :conf: prj_dtls.conf
+   :conf: prj.conf overlay-dtls.conf
```

Comment:
Add the quotes:

    :conf: "prj.conf overlay-dtls.conf"

Reply:
Thanks, fixing.